docs/user/intro/mkdocs.rst

```diff
@@ -26,37 +19,58 @@ specifying a python toolchain on Ubuntu, defining the location of the installati
     build:
       os: ubuntu-24.04
       tools:
-        python: "3"
-
-    python:
-      install:
-        - requirements: requirements.txt
+        python: 3
+      jobs:
+        pre_install:
+          - pip install mkdocs
+          - pip install mkdocs-material
 
     mkdocs:
       configuration: mkdocs.yml
 
 .. _MkDocs: https://www.mkdocs.org/
 .. _Material for MkDocs: https://squidfunk.github.io/mkdocs-material
 
+Configuring Material for MkDocs on Read the Docs
```

Comment:
This section is a bit confusing to me because we are already installing `mkdocs-material` using `build.jobs` in the previous YAML example.

Also, this section tells the user to create a `requirements.txt` file but it doesn't tell them how to configure Read the Docs to install these dependencies during build.

I would probably remove this section and just add a `seealso` with a sentence similar to "We recommend to pin your dependencies. See https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html". That way, we don't need to repeat ourselves on each doctool page.

Reply:
Ack -- I meant to revert the previous example and just use the requirements files.

Reply:
Either works. `build.jobs.post_install` is an quick copy&paste example that works without requiring too much else. Requirements is a better approach for the long term life of the project, tho. So 🤷🏼

Reply:
I think the copy/paste example is probably better. Updated the copy 💯